lib/sailsDbMigrate.js

```diff
@@ -77,15 +77,30 @@ function parseSailsConfig(sailsConfig) {
     throw new Error('Migrations not configured. Please setup ./config/migrations.js');
   }
 
-  var connectionName = sailsConfig.migrations.connection;
+  var connectionName = 
+    sailsConfig.migrations.datastore || 
+    // backwards compatibility: v0.10.x
+    sailsConfig.migrations.connection;
   if (!connectionName) {
-    throw new Error('connection missing from ./config/migrations.js');
+    throw new Error(
+      'Migrations not configured. Please specify either datastore (v1.x.x) or ' +
+      'connection (v0.10.x) in ./config/migrations.js'
+    );
+  }
+  
+  if (sailsConfig.connections) {
+    // backwards compatibility: v0.10.x
+    connection = sailsConfig.connections[connectionName]
+  } else if (sailsConfig.datastores) {
+    // v1.x.x
+    sailsConfig.datastores[connectionName]
```

Comment:
`connection = sailsConfig.datastores[connectionName]`

Reply:
Good catch, missed that after transforming it from the nested ternary operator.